src/meta/processors/admin/BalanceTask.cpp

```diff
@@ -152,6 +152,9 @@ void BalanceTask::invoke() {
                     ret_ = Result::FAILED;
                 } else {
                     status_ = Status::REMOVE_PART_ON_SRC;
+                    if (kv_ != nullptr) {
```

Comment:
We don't need to update the lateUpdateTime for each task. 

We could update the time when all balance plan finished.

Reply:
done!